suite/tests/CMakeLists.txt

```diff
@@ -3302,6 +3307,11 @@ if (BUILD_CLIENTS)
         "-simulator_type miss_analyzer -miss_count_threshold 5000 -miss_frac_threshold 0.25" "")
     endif ()
 
+    # Stress-test drcachesim with drstatecmp checks.
+    if (AARCH64)
```

Comment:
I know the fuzz app is a64-only -- but isn't this test available for x86?

Reply:
drcachesim with drstatecmp can tested for x86 for a different app. 
I added another drcachesim test with drstatecmp for the the ci_shared_app (so available for all archs).
Btw the code_api|sample.statecmp test is available for all archs.